pmd-apex/src/main/java/net/sourceforge/pmd/lang/apex/ast/ASTExpressionStatement.java

```diff
@@ -6,44 +6,18 @@
 
 import net.sourceforge.pmd.annotation.InternalApi;
 
-import com.google.summit.ast.statement.ExpressionStatement;
+import com.google.summit.ast.expression.Expression;
 
-public class ASTExpressionStatement extends AbstractApexNode.Single<ExpressionStatement> {
+public class ASTExpressionStatement extends AbstractApexNode.Single<Expression> {
 
     @Deprecated
     @InternalApi
-    public ASTExpressionStatement(ExpressionStatement expressionStatement) {
-        super(expressionStatement);
+    public ASTExpressionStatement(Expression expression) {
+        super(expression);
     }
 
     @Override
     public Object jjtAccept(ApexParserVisitor visitor, Object data) {
         return visitor.visit(this, data);
     }
-
-    /*
-    private int beginColumnDiff = -1;
-
-    @Override
-    public int getBeginColumn() {
-        if (beginColumnDiff > -1) {
-            return super.getBeginColumn() - beginColumnDiff;
-        }
-
-        if (getNumChildren() > 0 && getChild(0) instanceof ASTMethodCallExpression) {
```

Comment:
This seems like a workaround for something related to method calls. IWasthis a Jorje-specific thing?

Reply:
It seems like it was manually adjusting the range of the source location from Jorje's default value. The source locations are the most volatile thing when switching from Jorje to Summit (the start & end of each language structure is likely interpreted differently) so I assume that this is not relevant anymore.